module/spring-boot-amqp/src/test/java/org/springframework/boot/amqp/autoconfigure/RabbitAutoConfigurationTests.java

```diff
@@ -115,13 +115,15 @@
  * @author Phillip Webb
  * @author Scott Frederick
  * @author Yanming Zhou
+ * @author Eddú Meléndez
  */
 @ExtendWith(OutputCaptureExtension.class)
 class RabbitAutoConfigurationTests {
 
 	private final ApplicationContextRunner contextRunner = new ApplicationContextRunner()
 		.withConfiguration(AutoConfigurations.of(RabbitAutoConfiguration.class, SslAutoConfiguration.class))
-		.withClassLoader(new FilteredClassLoader("org.springframework.rabbit.stream")); // gh-38750
+		.withClassLoader(new FilteredClassLoader("org.springframework.rabbit.stream", "com.rabbitmq.client.amqp",
+				"org.springframework.amqp.rabbitmq.client")); // gh-38750
```

Comment:
I don't think a GH issue is relevant to your change.
Plus, I see you do the same in other place, but no GH issue number for whatsoever...

Reply:
I am leaving the GH issue reference from the original source code, related to the stream package. It was not added by my changes.

Reply:
Right. As I said: your change neglect an original goal of that issue.
Plus, when you fix to conditional on bean, then you would not need all these changes in tests.